src/view/items/element/specials/Input.js

```diff
@@ -5,4 +5,16 @@ export default class Input extends Element {
 		super.render( target, occupants );
 		this.node.defaultValue = this.node.value;
 	}
+	compare ( value, attrValue ) {
+		const comparator = this.getAttribute( 'value-comparator' );
+		if (comparator) {
+			if (typeof comparator === 'function') {
+				return comparator(value, attrValue);
+			}
+			if (value && attrValue) {
+				return value[comparator] == attrValue[comparator];
+			}
+		}
+		return value == attrValue;
+	}
```

Comment:
Any particular reason why the `==`?

Also, just throwing this in, in case you'd reconsider.

```js
return typeof comparator === 'function' ? comparator(value, attrValue)
                   : value && attrValue ? value[comparator] == attrValue[comparator]
                                        : value == attrValue
```